postgres-demo/README.md

```diff
@@ -5,15 +5,15 @@ web application using Apiary and [Spring Boot](https://spring.io/projects/spring
 We use Apiary's Postgres backend.
 To get started, let's first install some dependencies:
 
-    sudo apt install openjdk-11-jdk maven libatomic1
+    sudo apt install openjdk-11-jdk maven libatomic1 postgresql-client
 
 Next, let's compile Apiary. In the Apiary root directory, run:
 
     mvn -DskipTests package
 
 Then, let's start Postgres from a Docker image:
 
-    scripts/initialize_postgres_docker.sh
+    sudo scripts/initialize_postgres_docker.sh
```

Comment:
Maybe add a note here that we could run Docker without using sudo: https://docs.docker.com/engine/install/linux-postinstall/